lib/tasks/doc_facility.rake

```diff
@@ -0,0 +1,56 @@
+require 'open-uri'
+require 'csv'
+
+namespace :doc do
+  desc 'fill out doc_facility table and link records to status and profile'
+  task :doc_facilities do 
+    csv_path = 'lib/Prison.Facilities-Table.1.csv'
+    csv = []
```

Comment:
maybe we can update csv to be prison_names or something more descriptitive? When I see csv I think it's the literal entire csv rows